File: 02_ProgrammingFundamentals/_10_TextProcessing/_01_Lecture/_03_Substring.java
package _02_ProgrammingFundamentals._10_TextProcessing._01_Lecture;

import java.util.Scanner;

public class _03_Substring {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        String stringToRemove = scan.nextLine();
        String consoleString = scan.nextLine();
        
        // Remove the string from console
        while (consoleString.contains(stringToRemove)) {
            consoleString = consoleString.replaceFirst(stringToRemove, "");
        }
        
        // Print result
        System.out.println(consoleString);
        
        scan.close();
    }
}

File: 24_C#Fundamentals/_08_TextProcessing/_01_Lecture/_03_Substring/_03_Substring.cs
using System;
using System.Linq;

public class _03_Substring
{
    public static void Main(string[] args)
    {
        // Read input values
        string stringToRemove = Console.ReadLine();
        string consoleString = Console.ReadLine();

        // Remove the string from console
        while (consoleString.Contains(stringToRemove))
        {
            int index = consoleString.IndexOf(stringToRemove);
            if (index != -1)
            {
                consoleString = consoleString.Remove(index, stringToRemove.Length);
            }
        }

        // Print result
        Console.WriteLine(consoleString);
    }
}
